packages/core/src/utils/Resizer.ts

```diff
@@ -750,4 +756,20 @@ export default class Resizer {
 
     return box;
   }
+
+  hasPositionedParent(element: HTMLElement | null): boolean {
+    if (!element) return false;
+
+    let currentElement: HTMLElement | null = element;
+
+    while (currentElement) {
+      const position = window.getComputedStyle(currentElement).position;
+      if (position === 'relative' || position === 'absolute' || position === 'fixed' || position === 'sticky') {
+        return true;
+      }
+      currentElement = currentElement.parentElement;
+    }
+
+    return false;
+  }
```

Comment:
this looks expensive, can't we simply rely on [`offsetParent`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/offsetParent)?